wzdx/tools/uuid_tools.py

```diff
@@ -4,7 +4,7 @@
 import uuid
 
 
-CDOT_NAMESPACE_UUID = uuid.UUID('ef39d1dd-53ce-4651-9491-4bba7251a3c0')
+CDOT_NAMESPACE_UUID = uuid.UUID("ef39d1dd-53ce-4651-9491-4bba7251a3c0")
```

Comment:
What is this UUID used for? Is this something we should abstract to the env file?

Reply:
This is to seed all of the UUIDs with a shared namespace UUID. This allows all UUIDs generated to be predictable, while also being unique from other users. I moved this to an environment variable.